codegen/smithy-aws-typescript-codegen/src/main/java/software/amazon/smithy/aws/typescript/codegen/AddAwsRuntimeConfig.java

```diff
@@ -94,13 +94,11 @@ public void addConfigInterfaceFields(
             writer.writeDocs("Enables FIPS compatible endpoints.")
                     .write("useFipsEndpoint?: boolean | __Provider<boolean>;\n");
         }
-        if (!settings.getExperimentalIdentityAndAuth()) {
```

Comment:
removed condition on ID&A because other conditional locations for region do no longer include ID&A check.